cobalt/browser/cobalt_content_browser_client.cc

```diff
@@ -212,6 +230,10 @@ void CobaltContentBrowserClient::ConfigureNetworkContextParams(
   network_context_params->quic_user_agent_id = "";
   network_context_params->accept_language = GetApplicationLocale();
 
+  cobalt::browser::UpdateDefaultLocaleManager::GetInstance()->set_update(
```

Comment:
It's surprising to me to find the locale-setting code in
this are which looks related to network requests.